Sources/TableViewDriver.swift

```diff
@@ -232,16 +232,11 @@ extension TableViewDriver: UITableViewDataSource {
 
     /// :nodoc:
     public func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        var cell: UITableViewCell
-
-        if let cellViewModel = self.tableViewModel?[indexPath] {
-            cell = tableView.dequeueReusableCell(withIdentifier: cellViewModel.cellIdentifier, for: indexPath)
-            cellViewModel.applyViewModelToCell(cell)
-            cell.accessibilityIdentifier = cellViewModel.accessibilityFormat.accessibilityIdentifierForIndexPath(indexPath)
-        } else {
-            cell = UITableViewCell()
-        }
-
+        assert(self.tableViewModel != nil, "Table View Model has an invalid configuration")
+        let cellViewModel = self.tableViewModel![indexPath]!
```

Comment:
Same Q as above; another assert?

Also instead of assert, could `guard let` and throw a `fatalError` if no model is provided or no cell exists at index path?